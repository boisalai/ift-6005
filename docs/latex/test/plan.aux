\relax 
\providecommand\zref@newlabel[2]{}
\providecommand\babel@aux[2]{}
\@nameuse{bbl@beforestart}
\catcode `:\active 
\catcode `;\active 
\catcode `!\active 
\catcode `?\active 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand \oddpage@label [2]{}
\babel@aux{french}{}
\citation{hong2024next}
\citation{mohammadjafari2024natural}
\citation{zhu2024large}
\citation{gao2023text}
\@writefile{toc}{\contentsline {section}{\numberline {1}Description du problème}{2}{section.1}\protected@file@percent }
\newlabel{sec:probleme}{{1}{2}{Description du problème}{section.1}{}}
\newlabel{sec:probleme@cref}{{[section][1][]1}{[1][2][]2}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Revue de littérature}{2}{section.2}\protected@file@percent }
\newlabel{sec:revue}{{2}{2}{Revue de littérature}{section.2}{}}
\newlabel{sec:revue@cref}{{[section][2][]2}{[1][2][]2}}
\citation{biswal2024text2sql}
\citation{li2024can}
\citation{malekpour2024towards}
\@writefile{toc}{\contentsline {section}{\numberline {3}Approche proposée}{3}{section.3}\protected@file@percent }
\newlabel{sec:approche}{{3}{3}{Approche proposée}{section.3}{}}
\newlabel{sec:approche@cref}{{[section][3][]3}{[1][3][]3}}
\citation{biswal2024text2sql}
\@writefile{toc}{\contentsline {section}{\numberline {4}Données utilisées}{4}{section.4}\protected@file@percent }
\newlabel{sec:donnees}{{4}{4}{Données utilisées}{section.4}{}}
\newlabel{sec:donnees@cref}{{[section][4][]4}{[1][4][]4}}
\citation{li2024can}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Distribution de la complétude des colonnes de la base de données}}{5}{figure.1}\protected@file@percent }
\newlabel{fig:missing-values}{{1}{5}{Distribution de la complétude des colonnes de la base de données}{figure.1}{}}
\newlabel{fig:missing-values@cref}{{[figure][1][]1}{[1][4][]5}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Évaluation du système}{5}{section.5}\protected@file@percent }
\newlabel{sec:evaluation}{{5}{5}{Évaluation du système}{section.5}{}}
\newlabel{sec:evaluation@cref}{{[section][5][]5}{[1][4][]5}}
\bibstyle{plainnat}
\bibdata{refs}
\bibcite{biswal2024text2sql}{{1}{2024}{{Biswal et~al.}}{{Biswal, Patel, Jha, Kamsetty, Liu, Gonzalez, Guestrin, and Zaharia}}}
\bibcite{gao2023text}{{2}{2023}{{Gao et~al.}}{{Gao, Wang, Li, Sun, Qian, Ding, and Zhou}}}
\bibcite{hong2024next}{{3}{2024}{{Hong et~al.}}{{Hong, Yuan, Zhang, Chen, Dong, Huang, and Huang}}}
\bibcite{li2024can}{{4}{2024}{{Li et~al.}}{{Li, Hui, Qu, Yang, Li, Li, Wang, Qin, Geng, Huo, et~al.}}}
\@writefile{toc}{\contentsline {section}{\numberline {6}Tâches à faire}{6}{section.6}\protected@file@percent }
\newlabel{sec:taches}{{6}{6}{Tâches à faire}{section.6}{}}
\newlabel{sec:taches@cref}{{[section][6][]6}{[1][6][]6}}
\newlabel{sec:biblio}{{6}{6}{Tâches à faire}{Item.8}{}}
\newlabel{sec:biblio@cref}{{[section][6][]6}{[1][6][]6}}
\bibcite{malekpour2024towards}{{5}{2024}{{Malekpour et~al.}}{{Malekpour, Shaheen, Khomh, and Mhedhbi}}}
\bibcite{mohammadjafari2024natural}{{6}{2024}{{Mohammadjafari et~al.}}{{Mohammadjafari, Maida, and Gottumukkala}}}
\bibcite{zhu2024large}{{7}{2024}{{Zhu et~al.}}{{Zhu, Li, Cui, and Liu}}}
\gdef \@abspage@last{7}
