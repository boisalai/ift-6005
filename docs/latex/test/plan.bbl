\begin{thebibliography}{7}
\providecommand{\natexlab}[1]{#1}
\providecommand{\url}[1]{\texttt{#1}}
\expandafter\ifx\csname urlstyle\endcsname\relax
  \providecommand{\doi}[1]{doi: #1}\else
  \providecommand{\doi}{doi: \begingroup \urlstyle{rm}\Url}\fi

\bibitem[Biswal et~al.(2024)Biswal, Patel, Jha, Kamsetty, Liu, Gonzalez, Guestrin, and Zaharia]{biswal2024text2sql}
Asim Biswal, Liana Patel, Siddarth Jha, Amog Kamsetty, Shu Liu, Joseph~E Gonzalez, Carlos Guestrin, and Matei Zaharia.
\newblock Text2sql is not enough: Unifying ai and databases with tag.
\newblock \emph{arXiv preprint arXiv:2408.14717}, 2024.

\bibitem[Gao et~al.(2023)Gao, Wang, Li, Sun, Qian, Ding, and Zhou]{gao2023text}
Dawei Gao, Haibin Wang, Yaliang Li, Xiuyu Sun, Yichen Qian, Bolin Ding, and Jingren Zhou.
\newblock Text-to-sql empowered by large language models: A benchmark evaluation.
\newblock \emph{arXiv preprint arXiv:2308.15363}, 2023.

\bibitem[Hong et~al.(2024)Hong, Yuan, Zhang, Chen, Dong, Huang, and Huang]{hong2024next}
Zijin Hong, Zheng Yuan, Qinggang Zhang, Hao Chen, Junnan Dong, Feiran Huang, and Xiao Huang.
\newblock Next-generation database interfaces: A survey of llm-based text-to-sql.
\newblock \emph{arXiv preprint arXiv:2406.08426}, 2024.

\bibitem[Li et~al.(2024)Li, Hui, Qu, Yang, Li, Li, Wang, Qin, Geng, Huo, et~al.]{li2024can}
Jinyang Li, Binyuan Hui, Ge~Qu, Jiaxi Yang, Binhua Li, Bowen Li, Bailin Wang, Bowen Qin, Ruiying Geng, Nan Huo, et~al.
\newblock Can llm already serve as a database interface? a big bench for large-scale database grounded text-to-sqls.
\newblock \emph{Advances in Neural Information Processing Systems}, 36, 2024.

\bibitem[Malekpour et~al.(2024)Malekpour, Shaheen, Khomh, and Mhedhbi]{malekpour2024towards}
Mohammadhossein Malekpour, Nour Shaheen, Foutse Khomh, and Amine Mhedhbi.
\newblock Towards optimizing sql generation via llm routing.
\newblock \emph{arXiv preprint arXiv:2411.04319}, 2024.

\bibitem[Mohammadjafari et~al.(2024)Mohammadjafari, Maida, and Gottumukkala]{mohammadjafari2024natural}
Ali Mohammadjafari, Anthony~S Maida, and Raju Gottumukkala.
\newblock From natural language to sql: Review of llm-based text-to-sql systems.
\newblock \emph{arXiv preprint arXiv:2410.01066}, 2024.

\bibitem[Zhu et~al.(2024)Zhu, Li, Cui, and Liu]{zhu2024large}
Xiaohu Zhu, Qian Li, Lizhen Cui, and Yongkang Liu.
\newblock Large language model enhanced text-to-sql generation: A survey.
\newblock \emph{arXiv preprint arXiv:2410.06011}, 2024.

\end{thebibliography}
