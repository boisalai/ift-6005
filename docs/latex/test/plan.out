\BOOKMARK [1][-]{section.1}{\376\377\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n\000\040\000d\000u\000\040\000p\000r\000o\000b\000l\000\350\000m\000e}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000R\000e\000v\000u\000e\000\040\000d\000e\000\040\000l\000i\000t\000t\000\351\000r\000a\000t\000u\000r\000e}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000A\000p\000p\000r\000o\000c\000h\000e\000\040\000p\000r\000o\000p\000o\000s\000\351\000e}{}% 3
\BOOKMARK [1][-]{section.4}{\376\377\000D\000o\000n\000n\000\351\000e\000s\000\040\000u\000t\000i\000l\000i\000s\000\351\000e\000s}{}% 4
\BOOKMARK [1][-]{section.5}{\376\377\000\311\000v\000a\000l\000u\000a\000t\000i\000o\000n\000\040\000d\000u\000\040\000s\000y\000s\000t\000\350\000m\000e}{}% 5
\BOOKMARK [1][-]{section.6}{\376\377\000T\000\342\000c\000h\000e\000s\000\040\000\340\000\040\000f\000a\000i\000r\000e}{}% 6
